builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -252,6 +258,17 @@ func resourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
+		var privateIPAddress *string
+		for _, ipConfig := range *iface.IPConfigurations {
+			privateIPAddress = ipConfig.Properties.PrivateIPAddress
```

Comment:
Ok, if there is only 1, then we shouldn't range over right now

We should do something like this:

```
if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
 		var privateIPAddress *string
                 if iface.IPConfigurations[0] != nil {
                     privateIPAddress = iface.IPConfigurations[0].Properties.PrivateIPAddress
                 }
}
```

Then leave a note in place referencing that Azure SDK issue where only 1 NIC is allowed - thoughts?